### PR TITLE
New DevTools digest (Feb 2016)

### PR DESCRIPTION
**Pull request by pbakaus**
The latest DevTools digest with the best from the last couple weeks. Please review @paulirish @kaycebasques.

### CODE REVIEW

**Comment by Garbee** [comment]
Isn't the new Device Mode design shipping as well? Or will that get covered in the next digest closer to stable drop?

File: src/content/en/updates/posts/2016/02/devtools-digest-devtools-go-dark.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - pbakaus
+tags:
+  - devtools
+  - digest
+  - update
+description: "Learn how DevTools makes you type less with smarter Console autocomplete, how to edit @keyframe rules directly in the Styles pane, how to have fun with CSS Custom Variables and how to join the dark side."
+featured_image: /web/updates/images/2016/02/devtools-digest/dark_theme.png
+---
+
+<p class="intro">Learn how DevTools makes you type less with smarter Console autocomplete, how to edit @keyframe rules directly in the Styles pane, how to have fun with CSS Custom Variables and how to join the dark side.</p>
```
Comment (petele) [inline src/content/en/updates/posts/2016/02/devtools-digest-devtools-go-dark.markdown]: `@keyframe` rules should be wrapped in backticks

File: src/content/en/updates/posts/2016/02/devtools-digest-devtools-go-dark.markdown [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## A smarter autocomplete in your Console
+
+If you’re like me and many others, you type a command into the console to debug your app, see it not working, iterate and type it again, and again, and again. In order to help with that, we’re now autocompleting full statements you typed previously, like so:
+
+![Autocomplete](/web/updates/images/2016/02/devtools-digest/autocomplete.png)
+
+## Directly edit @keyframe rules in Styles pane
+
+When we introduced the animation inspector and easing editor to DevTools, it was limited to transitions because we never showed @keyframe-based CSS animations in the Style pane. I’m pleased to say that that’s now a thing of the past, so go wild! Check out our [video tweet](https://twitter.com/ChromeDevTools/status/694966453376675840) for a preview.
```
Comment (petele) [inline src/content/en/updates/posts/2016/02/devtools-digest-devtools-go-dark.markdown]: should `@keyframe-based` be backticketed

**Comment by petele** [comment]
@pbakaus a few minor tweaks if possible please

**Comment by petele** [comment]
PS - once those changes are made, go head and merge and let me know so we cna get a push done

**Comment by pbakaus** [comment]
All changes done!

**Comment by pbakaus** [comment]
@Garbee I wanna wait a little longer until it's actually stable..there were so many iterations still incoming ..

File: src/content/en/updates/posts/2016/02/devtools-digest-devtools-go-dark.markdown [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+![Autocomplete](/web/updates/images/2016/02/devtools-digest/autocomplete.png)
+
+## Directly edit @keyframe rules in Styles pane
+
+When we introduced the animation inspector and easing editor to DevTools, it was limited to transitions because we never showed `@keyframe`-based CSS animations in the Style pane. I’m pleased to say that that’s now a thing of the past, so go wild! Check out our [video tweet](https://twitter.com/ChromeDevTools/status/694966453376675840) for a preview.
+
+## Custom CSS Properties support
+
+![Custom CSS properties in DevTools](http://3.bp.blogspot.com/-gJn1NEz81YQ/VrDqiHlp85I/AAAAAAAAAvM/YtNyMyS0B-8/s1600/css-custom-properties-3.gif)
```
Comment (petele) [inline src/content/en/updates/posts/2016/02/devtools-digest-devtools-go-dark.markdown]: @pbakaus - see #2548, and please fix asap!!

**Pull request merged**